ide/hudson/src/org/netbeans/modules/hudson/Bundle.properties

```diff
@@ -17,7 +17,7 @@
 
 OpenIDE-Module-Display-Category=Base IDE
 OpenIDE-Module-Long-Description=\
-    Helps browse and manage installations of the Hudson continuous integration server (http://hudson-ci.org/). \
+    Helps browse and manage installations of the Jenkins continuous integration server (http://hudson-ci.org/). \
     http://wiki.netbeans.org/HudsonInNetBeans has more information.
 OpenIDE-Module-Name=Hudson
```

Comment:
@matthiasblaesing @lkishalmi I think we should also update `OpenIDE-Module-Name` in all involved modules so that users don't have to search for "hudson" to find it in the plugin manager, what do you think?

Reply:
@mbien Indeed, that would make sense. 

@rothardo sorry, I just noticed, that you pushed an update. FYI github does not inform when updates are force-pushed, so if you push an update like this it would be good to leave a comment. Could you have a look if the suggestion from @mbien also makes sense to you and update this also?

Reply:
does that imply to all the modules including [php/hudson.php](https://github.com/apache/netbeans/tree/master/php/hudson.php) ?

Reply:
> @mbien Indeed, that would make sense.
> 
> @rothardo sorry, I just noticed, that you pushed an update. FYI github does not inform when updates are force-pushed, so if you push an update like this it would be good to leave a comment. Could you have a look if the suggestion from @mbien also makes sense to you and update this also?

surely @matthiasblaesing I will keep in mind to leave a comment if I force push from next time.